_posts/rosi_bajari/2018-02-07-clinical-data-integration.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Integrating Clinical Annotation Features into the ICGC Data Portal"
+breadcrumb: true
+author: rosi_bajari
+date: 2018-02-07
+categories: rosi_bajari
+tags:
+    - ICGC
+    - data portal
+teaser:
+    info: In February 2018, we integrated new features focusing on clinical annotation of variants into the ICGC data portal.  Here, we will explore some of those features and how they can be used by researchers! 
+    image: rosi_bajari/clinical/post_thumb.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+As part of the ICGCs goal of creating comprehensive descriptions of cancer genomics, we are introducing a new integration of clinically relevant data to the ICGC data portal from [CIVIC](https://civic.genome.wustl.edu/), a database of curated, clinically relevant cancer variants, and [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/), a database of alleles and their phenotype relationship.  CIVIC curated variants are reviewed regularly by curators and editors and help link mutations to published supporting material as well as targeted therapeutics.   ClinVar annotates the clinical significance of genetic variants based on aggregated results of published evidence.  
+
+# How does this new clinical data fit into the ICGC data portal ? 
+The ICGC has collected a massive volume of cancer data.  To date, there are 76 participating cancer projects with 17,440 donors with molecular data, with a whopping 68 million individual mutations!  Annotating, the process of identifying data features, helps researchers sort through all 68 million mutations to help answer scientific inquiries.  Through our new clinical integration, 447 Pathogenic or likely pathogenic mutations have been identified. 
+
+<center>
+  <figure>
+      <img src="{{site.urlimg}}rosi_bajari/clinical/gene_pyrmid.png"/>  
+      <figcaption>Data pyramiding - sorting from everything to the most important things</figcaption>
+  </figure>
+</center>
+
+Now that we have all the data, the next questions is "How can scientific community members use this information?". Let’s examine how researchers or clinicians could use this information to help answer these questions.   
+
+# Finding significant mutations
+
+One interesting question a cancer researcher could ask would be: How many clinically significant mutations in the KRAS gene from the [ICGC PCAWG](http://docs.icgc.org/pcawg/) (PanCancer Analysis of Whole Genomes) donors are there?  Using our new clinical significance and evidence facets in Advanced Search, there are 17 different mutations across 282 donors (query accessible [here](https://icgc.org/Zuh)). This interesting discovery can lead to more questions to investigate like how effectively a clinically significant mutation can predict survival rate, or what pathways the mutation might affect that causes certain cancerous symptoms.  
```

Comment:
Maybe adding a comma between investigate and like "to more questions to investigate, like how effectively"

Reply:
It will not longer accept my push bc of permissions again :cold_sweat:. Anyways, I don't want to waste more time debugging ssh keys today. Can we merge without more edits?